### PR TITLE
Fix installation

### PR DESCRIPTION
**Pull request by Jibec**
previous version isn't working yet, some issue with path

I aslo added a security fix for nginx and set my name as maintainer

### CODE REVIEW

File: conf/config.ini [hunk 1]
```diff
@@ -48,7 +48,7 @@ python_exec=
 ; Overridden by the KRESUS_WEBOOB_DIR environment variable, if it's set.
 ; Example:
 ; srcdir=/home/ben/code/weboob
-srcdir=__FINALPATH__/venv/bin/
+srcdir=__FINALPATH__/venv/lib/python2.7/site-packages/
```
Comment (ZeHiro) [inline conf/config.ini]: As discussed on framagit, I would suggest to set : 
- unset `srcdir` (leave it empty)
- set `python_exec=__FINALPATH__/venv/bin/python` 
Setting the python exe to the one in the virtual env will automatically set the PATH correctly (and could include other environnement variables which are not included with `srcdir`).

File: README.md [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
+  * [ ] Email admin about installation with ynh_send_readme_to_admin helper
+
+## Links
 
-**More information on the documentation page:**
-https://framagit.org/bnjbvr/kresus
-https://kresus.org
-https://yunohost.org/
+ * Report a bug about this package: https://github.com/YunoHost-Apps/kresus_ynh
+ * Report a bug about Kresus itself: https://framagit.org/bnjbvr/kresus
```
Comment (ZeHiro) [inline README.md]: We just moved the repo to `https://framagit.org/kresusapp/kresus`

File: README.md [hunk 3]
```diff
@@ -1,14 +1,28 @@
...
+ * Documentation: https://docs.funkwhale.audio
```
Comment (ZeHiro) [inline README.md]: What is this page ?

**Comment by ZeHiro** [comment]
Thanks for this update of the package. Note that the repo on github has also changed: `https://github.com/kresusapp/kresus` (even if there is currently a redirection from @bnjbvr repo)

**Comment by Jibec** [comment]
Everything should be fixed now :+1:

**Comment by ZeHiro** [comment]
It works out of the box, though I could not test the install at the root of a sub-domain.

**Comment by LowMem** [comment]
I merge this PR, even if 

> the install at the root of a sub-domain.
could not be tested: the 1st line of the README clearly states to not use it in production… Hopefully this statement will be changed after some testing and a new release !?! :rofl:

Thanks all for the great work !
:heart:

**Comment by Jibec** [comment]
thanks, I do confirm it works fine with root domain installation on a virtual machine :)

**Comment by LowMem** [comment]
Revert done and pushed. :-)

**Pull request merged**